package/lean/xray/Makefile

```diff
@@ -277,12 +276,11 @@ endif
 endef
 
 define Package/$(PKG_NAME)/install
-	$(INSTALL_DIR) $(1)/usr/bin/xray
-
-	$(INSTALL_BIN) $(GO_PKG_BUILD_BIN_DIR)/xray $(1)/usr/bin/xray
-
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(GO_PKG_BUILD_BIN_DIR)/xray $(1)/usr/bin
```

Comment:
With building scripts from v2fly official, main program and data files should be installed in the same directory.
And, to make it compatible with other LuCI App, we should keep the installing directory as-is.

Reply:
We should use the correct location
[Filesystem Hierarchy Standard (FHS)](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)

Reply:
This breaks both the standards from v2fly and other LuCI apps.

You should make these changes upstreamed before modifying here.

Reply:
I must point that again, keep installing directory as-is.
V2Fly only read data files from the same directory with hard-coded file name.